test/server/cards/01_SOR/bases/SecurityComplex.spec.js

```diff
@@ -0,0 +1,34 @@
+describe('SecurityComplex', function() {
+    integration(function() {
+        describe('Security Complex\'s ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        base: 'security-complex',
+                    },
+                    player2: {
+                        groundArena: [
+                            { card: 'frontier-atrt', damage: 1 },
+                            'wampa'
+                        ],
+                        leader: { card: 'boba-fett#daimyo', deployed: true, damage: 1 }
```

Comment:
Just for clarity, could we remove the parts of this test setup that aren't relevant: the Boba leader and the damage on frontier at-rt.